redfish-core/lib/oem/ibm/pcie_topology_refresh.hpp

```diff
@@ -46,6 +48,85 @@ inline void
         "PCIeTopologyRefresh");
 }
 
+/**
+ * @brief PCIe Topology Refresh monitor. which block incoming request
+ *
+ * @param[in] timer       pointer to steady timer which block call
```

Comment:
Nit: which blocks further calls to refreshProperty? or blocks all further calls?